corgi/tasks/sca.py

```diff
@@ -73,8 +73,23 @@ def cpu_software_composition_analysis(build_id: int):
     logger.info("Started software composition analysis for %s", build_id)
     software_build = SoftwareBuild.objects.get(build_id=build_id)
 
-    # Get root component for this build; fail the task if it does not exist.
-    root_component = Component.objects.filter(software_build=software_build).root_components().get()
+    component_qs = Component.objects.filter(software_build=software_build)
+    try:
+        # Get root component for this build.
+        root_component = component_qs.root_components().get()
+    except Component.DoesNotExist as exc:
+        # None of the components were root components
+        module_qs = component_qs.filter(type=Component.Type.RPMMOD)
```

Comment:
RPMMOD components are just repackaging of existing RPMS. Therefore I think we should skip SCA on those components.